include/modules/i3.hpp

```diff
@@ -89,6 +93,8 @@ namespace modules {
     bool m_pinworkspaces{false};
     bool m_strip_wsnumbers{false};
     bool m_fuzzy_match{false};
+    bool m_show_icons{false};
+    string m_icons_side{"right"};
```

Comment:
Could we use an enum for this?

This could also eliminate the need for `show-icons`, if the possible values are `none`, `left`, and `right` (similar to the tray).